index.php

```diff
@@ -21,8 +21,7 @@
 if (
     $kirbyVersion !== null &&
     (
-        version_compare($kirbyVersion, '3.5.5', '<') === true ||
-        version_compare($kirbyVersion, '3.6.0-alpha', '>=') === true
+        version_compare($kirbyVersion, '3.5.5', '<') === true
```

Comment:
Should probably test against lower than 3.6 now. But actually a great way to prevent a plugin form being installed with incompatible lower Kirby versions!